pyk/kit-tests/test-kit.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+summary_script="$1" ; shift
+source ../kit-shell
+cd $(dirname ${summary_script})
+source $(basename ${summary_script})
```

Comment:
Are we `cd ...` so that the kit-script can just have shorter relative ptahs in `kit-load ...`, or for some other reason?

Reply:
Yeah, just so that we do not need to write out the path several times in the test script.